feather/protocol/src/packets/server/play.rs

```diff
@@ -1162,7 +1162,9 @@ packets! {
     }
 
     EntityProperties {
-        __todo__ LengthInferredVecU8;
+        entity_id VarInt;
+        properties LengthPrefixedVec<EntityProperty>;
```

Comment:
This needs to be an `IntPrefixedVec` which you need to implement first, like described on Discord.
Kinda forgot to comment that here and it might not have been that clear in Discord what I mean specifically.